_docs/backenders/strategy-2021/roles/hiring.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: -
+
+
+
+***Note:***
+
+2 people Identify the hiring related tasks to be managed between different roles:
+- Capacity
+- Strategy
+- Mentorship Coordination
+
+## Responsibilities
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions
+          
+          Priority: HIGH priority for all   
+          Identified as responsibility to be shared with Capacity Role 
+
+
+- Liaising with P&C about BE team's hiring needs - contacting P&C ahead of time, updating on BE decisions 
+  
+          Priority: HIGH priority for all   
+          Identified as responsibility to be shared with Capacity, Strategy, M.C roles
+
+
+- Distributing tasks: reviewing candidates, attending interviews
+        
+          Priority: Average Medium priority. (2 Low, 1 Medium, 1 High)
+          Agn and Andr are happy with the way that Sebastion is managing his part of this responsibility
+          Andr thinks this responsibility should me managed by each developer
+
+
+- Liaising with P&C about onboarding of new BE members
+       
+          Priority: Medium (1 Low, 1 Medium, 2 High)
+          Identified as responsability to be shared with Capacity, Strategy, M.C roles
+
+
+- Collaborating with P&C on job ads text
+
+          Priority: Medium-High ( 2 Medium, 2 High) 
+          Identified as responsability to be shared with Capacity, Strategy roles
+          Andr's note: Everyone should be encouraged to take part, with ownership on the Hiring and Strategy roles
+
+
+- Preparing candidate profile descriptions for outsourcing needs
+  
+          Priority: Medium ( 3 Medium, 1 High) 
+          Identified as responsability to be shared with Capacity Role
+          Andr's note: This responsibility should me managed by each developer, with advice from at least another team member
+                       We should try to keep this rather standard, once we can get there of course
+
+
+## Impact
+
```

Comment:
Reduce administrative burden on the BE team
Improve timeliness of starting hiring processes

Reply:
Added 88912f620275e49546b134901c22eb14d9283e3f
Thanks 😃